realio/chain.json

```diff
@@ -97,12 +97,26 @@
           "version": "0.34"
             },
           "ibc_go_version": "6.1.1",
+          "next_version_name": "v0.8.3"
+        },
+        {
+          "name": "v0.8.3",
+          "recommended_version": "v0.8.3",
+          "compatible_versions": [
+            "v0.8.2","v0.8.3"
```

Comment:
if v0.8.2 is compatible with v0.8.3, then we simply update the existing v0.8.2 version object (but it looks like you've added a new object, which could be confusing)

Reply:
yeah v0.8.2 is compatible with v0.8.3

Reply:
Sorry, this was never resolved. There is still a separate version object for v0.8.2 that should be removed
<img width="296" alt="image" src="https://github.com/cosmos/chain-registry/assets/95667791/f5511695-bba4-4dbd-89a3-e232b2b007e4">

Reply:
https://github.com/cosmos/chain-registry/pull/2663/commits/eadf6aeb331e028e299dd749c39da976ba4204c7